scripts/dev_changelog.py

```diff
@@ -9,24 +10,31 @@
 see it.
 
 Usage::
+
     $ ./scripts/dev_changelog.py <token> <revision range> <output_file>
 
 The output is utf8 rst.
 
 Dependencies
 ------------
+
 - gitpython
 - pygithub
 
 Examples
 --------
+
 From the bash command line with $GITHUB token::
-    $ ./scripts/dev_changelog.py $GITHUB v0.3.0..v0.4.0 -o 0.4.0-changelog.rst
+
+    $ ./scripts/dev_changelog.py $GITHUB v0.3.0..HEAD -t v0.4.0 -o 0.4.0-changelog.rst
```

Comment:
I like this! I can update the GitHub Wiki accordingly.

One thing I'll add is that `..HEAD` portion of `v0.3.0..HEAD` seems unnecessary with this. I think the revision_range logic can be modified

Reply:
Agreed. I'll have to leave for a while, feel free to simply push to my branch if you get to refactor it in the meantime.